flycheck-mix.el

```diff
@@ -87,13 +90,7 @@
           (shell-quote-argument (flycheck-mix-project-root))
           ))
 
-;;;###autoload
-(defun flycheck-mix-setup ()
-  "Setup Flycheck for Elixir."
-  ;; it doesn't make sense to check buffer on idle change
-  ;; only saved files will be checked by mix
-  (setq flycheck-check-syntax-automatically '(mode-enabled save))
-  (add-to-list 'flycheck-checkers 'elixir-mix))
+(add-to-list 'flycheck-checkers 'elixir-mix)
```

Comment:
thanks, wasn't sure about that. I was following https://github.com/flycheck/flycheck-mercury/blob/master/flycheck-mercury.el#L222 which was probably not a good example :smile: